pastas/timeseries_utils.py

```diff
@@ -518,3 +519,35 @@ def pandas_equidistant_asfreq(series: Series, freq: str) -> Series:
         .squeeze()
     )
     return spandas
+
+
+def resample(
+    series: Series, freq: str, closed: str = "right", label: str = "right", **kwargs
+) -> Resampler:
+    """Wrapper around the pandas resample function with logical some Pastas defaults.
+       In Pastas, we assume the timestamp is at the end of the period that
+       belongs to each measurement. For more information on this read the
+       example notebook on preprocessing time series.
+
+    Parameters
+    ----------
+    series : Series
+        time series
+    freq : str
+        frequency string
+    closed: str, default 'right'
+        which side/end of bin interval is closed
+    label: str, default 'right'
+        which bin edge label to label bucket with
+    **kwargs: dict
+
+    Returns
+    -------
+    Resampler
+        pandas Resampler object which can be manipulated using methods such as:
+        '.interpolate()', '.mean()', '.max()' etc. For all options see:
+        https://pandas.pydata.org/docs/reference/resampling.html
+
+    """
+
+    return series.copy().resample(freq, closed=closed, label=label, **kwargs)
```

Comment:
Is this copy necessary?

Reply:
Don't know. I just do it as a habbit. We do it for the other `timeseries_utils` functions as well, e.g.:
https://github.com/pastas/pastas/blob/730251140dab2392b88dc15050f6c2ed006bf4b9/pastas/timeseries_utils.py#L453

Reply:
The one you link is probably not necessary either, but in other cases we do modify something about the series, which makes the copy necessary to ensure we leave the original data untouched. 

We can remove this one though.